apps/meteor/app/api/server/api.ts

```diff
@@ -678,18 +713,26 @@ export class APIClass<TBasePath extends string = ''> extends Restivus {
 								responseTime: Date.now() - startTime,
 							});
 						} catch (e: any) {
-							const apiMethod: string =
-								{
-									'error-too-many-requests': 'tooManyRequests',
-									'error-unauthorized': 'unauthorized',
-								}[e.error as string] || 'failure';
-
-							result = (API.v1 as Record<string, any>)[apiMethod](
-								typeof e === 'string' ? e : e.message,
-								e.error,
-								process.env.TEST_MODE ? e.stack : undefined,
-								e,
-							);
+							result = ((e: any) => {
+								switch (e.error) {
+									case 'error-too-many-requests':
+										return API.v1.tooManyRequests(typeof e === 'string' ? e : e.message);
+									case 'unauthorized':
+									case 'error-unauthorized':
+										if (applyBreakingChanges) {
+											return API.v1.unauthorized(typeof e === 'string' ? e : e.message);
+										}
+										return API.v1.forbidden(typeof e === 'string' ? e : e.message);
```

Comment:
Isn't this in a reversed order?

Reply:
It's a bit complex, but the PR is exactly about these changes.

You need to keep 2 or 3 things in mind at the same time:

- We used to call the unauthorized method (which would be correct) but passed the status code for forbidden. -- With the renaming of methods, even though it's incorrect, the natural approach is to start calling forbidden to avoid any breaking changes.